### PR TITLE
[Refactor:System] Fixed broken sym links

### PR DESCRIPTION
**Pull request by saumyaborwankar**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### CODE REVIEW

File: dockerfiles/submitty/tutorial/database_client/Dockerfile [hunk 1]
```diff
@@ -199,6 +199,9 @@ RUN set -eux; \
 	pip --version
 
 # Install additional packages via pip
+RUN ln -s /usr/local/bin/python3 /usr/bin/python3 \
```
Comment (bmcutler) [inline dockerfiles/submitty/tutorial/database_client/Dockerfile:202]: would it be better to add /usr/local/bin to the path?  (would that solve the problem?)

Comment (saumyaborwankar) [reply dockerfiles/submitty/tutorial/database_client/Dockerfile:202]: The tutorial wants /usr/bin/python3 to be the location for python3 for some reason. The path already has /usr/local/bin.

Comment (bmcutler) [reply dockerfiles/submitty/tutorial/database_client/Dockerfile:202]: Sorry I got that backwards
Is /usr/bin on the path?

Comment (bmcutler) [reply dockerfiles/submitty/tutorial/database_client/Dockerfile:202]: I'd rather standardize the tutorial than hack a symlink

Comment (saumyaborwankar) [reply dockerfiles/submitty/tutorial/database_client/Dockerfile:202]: image
This is the PATH inside docker container tutorial:db_client

**Pull request merged**